### ISSUE TITLE
Support Raw string literal text 

### ISSUE THREAD

**Issue by niltor** [Swiftly1/CsharpToColouredHTML#24]
if the c# code like:
```csharp
var xml = $"""

        <element attr="{content}">
            <body>
            </body>
        </element>
        """;
```

the result content like:
`var xml = $"""        <element attr="{content}">            <body>            </body>        </element>        """;`

**Comment by Swiftly1** [comment]
I will look into that, thanks

**Comment by Swiftly1** [comment]
Should be fixed in version 1.0.44

Let me know if it works for ya, so I'll close this

![obraz](https://github.com/Swiftly1/CsharpToColouredHTML/assets/77643169/75ea05da-c7fb-4b90-a020-d8870a74a0b9)
